ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.sass

```diff
@@ -1,17 +1,21 @@
-@import "../../../../styles/colors"
+@import '../../../../styles/colors'
+@import '../../../../styles/fonts-constants'
+@import '../../../../styles/constants'
+@import '../../../../styles/mixins'
 .btn-group
   position: fixed
   z-index: 3 // 3, because <mat-sidenav> default z-index:2
   bottom: 0
   width: 100%
-  .passanger-btn
+  .passenger-btn
```

Comment:
passenger-btn has the same styles as driver-btn. Pls remove those classes and add role-chooser-btn

Reply:
fixed